tests/Unit/Domain/Creators/Test_Sphere.cpp

```diff
@@ -537,6 +600,13 @@ void test_sphere() {
     CAPTURE(which_wedges);
     CAPTURE(time_dependent);
     CAPTURE(with_boundary_conditions);
+    // If we aren't time dependent, just set the hard coded option to false to
+    // avoid ambiguity. If we are time dependent but we're filling the interior,
+    // then we can't use hard coded options
+    if ((not time_dependent) or (time_dependent and fill_interior)) {
```

Comment:
[optional] The second `time_dependent` is redundant.  If it were false we would never get there.